modules/sno-du-configuring-the-klusterlet-addon.adoc

```diff
@@ -0,0 +1,36 @@
+// Module included in the following assemblies:
```

Comment:
Only applicable if the cluster is ACM managed

Reply:
Should this topic be removed?

Reply:
@browsell Should this be removed?

Reply:
@browsell Should this be removed?

Reply:
@juphoff Please verify Brent's comments- if the file should be removed or the note about ACM added (line 9)